imap/xapian_wrap.cpp

```diff
@@ -2038,14 +2061,18 @@ int xapian_query_run(const xapian_db_t *db, const xapian_query_t *qq, int is_leg
 
             /* ignore documents with no cyrusid.  Shouldn't happen, but has been seen */
             if (cyrusid.length() != 43) {
-                syslog(LOG_ERR, "IOERROR: Xapian: zero length cyrusid for document id %u in index files %s",
-                                d.get_docid(), db->paths->c_str());
+                // XXX paths seems to be plural here?
```

Comment:
Great, thanks for the clarification.  I won't get rid of it in this PR to keep the scope narrow (esp since I've needed to add it to another xsyslog myself), but we can always refactor it later.